packages/oracle-manager/contracts/utils/ReducerNodeLibrary.sol

```diff
@@ -0,0 +1,135 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.8.0;
+
+import "../storage/NodeFactoryStorage.sol";
+
+library ReducerNodeLibrary {
```

Comment:
this library is a great example for why I think this strategy for nodes works so well. its looking really good, even has a `quickSort` function which, tbh, might be overkill for solidity, but its great